### ISSUE TITLE
Move CSS to LESS or SASS

### ISSUE THREAD

**Issue by ghost** [ScalaConsultants/Learn-JS#13]
As we are trying to learn most innovative and cutting edge technologies I would like to propose moving from CSS to one of style sheet processors - either LASS or SASS. As I have no prior experience with neither I do not favor any of them.

**Comment by pjazdzewski1990** [comment]
Would you prefer to do it before we responsive design or after? I'm not how to introduce that as valid CSS should also be valid LASS/SASS. I feel like having more material to work with might make it easier to learn something.

**Comment by ghost** [comment]
We can do it while making the app responsive. That actually makes more sense. I have also found:

http://stylus-lang.com

**Comment by burczu** [comment]
yes, LESS is now a little bit obsolete - my I advice is to use Sass or Stylus, but I think that Sass would be the better option because is much more popular and it's good to know it
